schematics/dashboard/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.__styleext__

```diff
@@ -0,0 +1,17 @@
+.grid-container {
+  margin: 20px;
+}
+
+mat-card {
```

Comment:
Why absolute positioning?

Reply:
I couldn't come up with a good way to make the card 100% height/width with 15px margin. Open to ideas if you have any.

Reply:
`calc(100% - 15px)`?